application/org.openjdk.jmc.flightrecorder.flameview-java/src/main/java/org/openjdk/jmc/flightrecorder/flameviewjava/Messages.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022, Datadog, Inc. All rights reserved.
```

Comment:
Needs to be an Oracle/Datadog dual copyright. Check other Datadog provided files for examples.